src/main/java/address/controller/PersonOverviewController.java

```diff
@@ -117,7 +119,10 @@ private void initialize() {
      */
     @FXML
     private void handleDeletePersons() {
-        final List<ReadOnlyViewablePerson> selected = personListView.getSelectionModel().getSelectedItems();
+        final List<ReadOnlyViewablePerson> selected = new ArrayList<>(personListView.getSelectionModel()
+                                                                                    .getSelectedItems());
+        selected.removeIf(p -> !(p instanceof  ReadOnlyViewablePerson));
```

Comment:
As mentioned,  I notice that get selectedItems from SelectionModel will have item that is not ReadOnlyPerson. Therefore causing the NPE at #326.